new_requirements.txt

```diff
@@ -0,0 +1,221 @@
+pytest-mock @ file:///C:/Windows/TEMP/abs_d625af48-56cc-4c95-98c6-464b9f29a4ady1r7z176/croots/recipe/pytest-mock_1655844719340/work
+pytest-xdist @ file:///opt/conda/conda-bld/pytest-xdist_1651785087298/work
+python-dateutil @ file:///tmp/build/80754af9/python-dateutil_1626374649649/work
+pythreejs==2.3.0
+pytz @ file:///C:/Windows/TEMP/abs_90eacd4e-8eff-491e-b26e-f707eba2cbe1ujvbhqz1/croots/recipe/pytz_1654762631027/work
+pyvista==0.35.1
+pyviz-comms==2.2.0
+pywin32==303
+pywinpty @ file:///C:/ci_310/pywinpty_1644230983541/work/target/wheels/pywinpty-2.0.2-cp310-none-win_amd64.whl
+PyYAML==6.0
+pyzmq==22.3.0
+regex @ file:///C:/ci/regex_1648429714499/work
+requests @ file:///C:/ci/requests_1656443888572/work
+s3fs==2022.5.0
+scipy==1.8.1
+scooby==0.5.12
+Send2Trash @ file:///tmp/build/80754af9/send2trash_1632406701022/work
+Shapely==1.7.0
+six @ file:///tmp/build/80754af9/six_1644875935023/work
+snakeviz==2.1.1
+snowballstemmer @ file:///tmp/build/80754af9/snowballstemmer_1637937080595/work
+sortedcontainers==2.4.0
+soupsieve @ file:///tmp/build/80754af9/soupsieve_1636706018808/work
+Sphinx @ file:///opt/conda/conda-bld/sphinx_1643644169832/work
+sphinxcontrib-applehelp @ file:///home/ktietz/src/ci/sphinxcontrib-applehelp_1611920841464/work
+sphinxcontrib-devhelp @ file:///home/ktietz/src/ci/sphinxcontrib-devhelp_1611920923094/work
+sphinxcontrib-htmlhelp @ file:///tmp/build/80754af9/sphinxcontrib-htmlhelp_1623945626792/work
+sphinxcontrib-jsmath @ file:///home/ktietz/src/ci/sphinxcontrib-jsmath_1611920942228/work
+sphinxcontrib-qthelp @ file:///home/ktietz/src/ci/sphinxcontrib-qthelp_1611921055322/work
+sphinxcontrib-serializinghtml @ file:///tmp/build/80754af9/sphinxcontrib-serializinghtml_1624451540180/work
+stack-data==0.2.0
+tblib==1.7.0
+tenacity==8.0.1
+terminado @ file:///C:/ci/terminado_1644312135040/work
+testpath @ file:///C:/Windows/TEMP/abs_23c7fa33-cbb9-46dc-b7c5-590c38e2de3d4bmbngal/croots/recipe/testpath_1655908553202/work
+toml @ file:///tmp/build/80754af9/toml_1616166611790/work
+tomli @ file:///C:/Windows/TEMP/abs_ac109f85-a7b3-4b4d-bcfd-52622eceddf0hy332ojo/croots/recipe/tomli_1657175513137/work
+toolz==0.11.2
+tornado==6.1
+tqdm @ file:///C:/ci/tqdm_1650618044857/work
+traitlets==5.1.1
+traittypes==0.2.1
+typed-ast @ file:///C:/ci_310/typed-ast_1642093438825/work
+typing_extensions @ file:///opt/conda/conda-bld/typing_extensions_1647553014482/work
+ukkonen @ file:///D:/bld/ukkonen_1643058610911/work
+urllib3 @ file:///C:/ci/urllib3_1650621844306/work
+virtualenv @ file:///C:/ci_310/virtualenv_1642406296278/work
+vtk==9.2.0rc1
+wcwidth==0.2.5
+webencodings==0.5.1
+widgetsnbextension==3.6.1
+win-inet-pton @ file:///C:/ci_310/win_inet_pton_1642658466512/work
+wincertstore==0.2
+wrapt @ file:///C:/ci_310/wrapt_1642094330997/work
+wslink==1.6.6
+xarray==2022.3.0
+xarray-datatree==0.0.4
+yarl==1.7.2
+zarr @ file:///tmp/build/80754af9/zarr_1619738198884/work
+zict==2.2.0
+zipp @ file:///C:/ci/zipp_1652255909324/work
```

Comment:
It appears that this requirements file is a pip freeze that is very specific to your computer. Please create a more slim down version, where you're only listing the most necessary packages. See https://github.com/OSOceanAcoustics/echopype/blob/main/requirements.txt for example. Thanks!